src/gen/java/org/mghpcc/aitelemetry/model/baremetalnetwork/BareMetalNetworkPageGen.java

```diff
@@ -127,11 +127,7 @@ public Future<BareMetalNetworkPageGen<DEV>> promiseDeepBareMetalNetworkPage() {
 		Promise<Void> promise2 = Promise.promise();
 		promiseBareMetalNetworkPage(promise2);
 		promise2.future().onSuccess(a -> {
-			super.promiseDeepBareMetalNetworkGenPage(siteRequest_).onSuccess(b -> {
-				promise.complete(this);
-			}).onFailure(ex -> {
-				promise.fail(ex);
-			});
```

Comment:
This difference is due to a bug in the page generation after indexing the application code in the search engine for the first time. I am working on a fix for this.

Reply:
@tzumainn for now, just touch this file `touch /opt/app-root/src/ai-telemetry/src/main/java/org/mghpcc/aitelemetry/model/baremetalnetwork/BareMetalNetwork.java` and then it should fill in these gaps, and then amend your commit. Then I will make some PR with the BareMetalOrder.java I've been working on.

Reply:
@computate done!